tox.ini

```diff
@@ -431,7 +431,7 @@ deps =
     falcon-v2: falcon~=2.0
     falcon-v3: falcon~=3.0
     # TODO: update to 4.0 stable when out
-    falcon-v4: falcon==4.0.0b3
+    falcon-v4: falcon==4.0.0b4
```

Comment:
I am assuming this is needed because `falcon-latest` excludes pre-release versions? Is there any way we can automatically grab the latest pre-release version?